src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -162,6 +187,13 @@ private IReadOnlyList<Links> CreateArtistLinks(ArtistDto dto, string? fields)
                                        $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
         links.Add(link);
 
+        link = new Links(Url.RouteUrl("GetArtistByWalletAddress", new { dto.UserWalletAddress }),
+                         "artist_walletAddress",
```

Comment:
This value should be `get_userWalletAddress`

Reply:
Resolved